pyproject.toml

```
@@ -30,8 +30,9 @@ dependencies = [
     "pandas>=2.0,<=2.2.2",
     "pybtex",
     "pybtex-apa-style",
-    "robustica>=0.1.3",
+    "robustica>=0.1.3, <=0.1.3",
```

Could you pin the version?
```suggestion
    "robustica==0.1.3",
```

---

Will this work with the version update bot?  My assumption was we needed a `<=` to give the bot something to update, but I don't really know.

---

dependabot should just bump the version. So if robustica releases 0.1.4, we'd get a PR changing it to `robustica==0.1.4`.

---

Is that what we want or would "robustica>=0.1.3, <=0.1.4", be more in line with our other modules? I'm mostly neutral on this.